app/dashboard/templates/bounty/details.html

```diff
@@ -58,50 +58,70 @@ <h4 id="title" class="font-title"></h4>
               <div class="col-12">
                 <div id="status"></div>
               </div>
-              <div class="col-12 bounty-info">
-                <div id="timer">
-                  <div>
-                    <span class="bounty-info-heading timeleft">{% trans "Time left" %} </span>
-                    <span class="bounty-info-text" id="expires_date"></span>
+              <div id="bounty-info-row" class="row">
+                  <div class="col-12 col-sm-6 col-xl-3 bounty-info">
```

Comment:
This indentation needs fixed. 2-space indentation throughout *.html files please :D